src/components/TodoListItem.js

```diff
@@ -6,10 +6,10 @@ const TodoListItem = ({ todo, onRemoveTodo }) => {
 
     return (
         <li className={style.ListItem}>
-            {todo.fields.Title}&nbsp;
             <button type='button' onClick={() => onRemoveTodo(todo.id)}>
```

Comment:
this `button` element needs some form of alternative text for accessibility purposes, since there is no text content (icon only)

`title` or `aria-label` would suffice

see documentation: https://dequeuniversity.com/rules/axe/3.5/button-name